spark3.0/build.sbt

```diff
@@ -38,18 +38,21 @@ kafkaVersion := s"2.2.1-${distro.value}"
 // java.lang.NoClassDefFoundError: com/splicemachine/access/HConfiguration
 // https://stackoverflow.com/a/46763742/1305344
 
-val excludedDeps = Seq(
+val excludedDepsNonSpark = Seq(
   ExclusionRule(organization = "org.xerial.snappy", name = "snappy-java"),
   ExclusionRule(organization = "tomcat", name = "jasper-compiler"),
-  // FIXME Somehow 2.2.0 is pulled down
-  ExclusionRule(organization = "org.apache.spark"),
   // Added later separately
   ExclusionRule(organization = "com.splicemachine", name = "scala_util"),
   ExclusionRule(organization = "javax.ws.rs", name = "javax.ws.rs-api"),
   ExclusionRule(organization = "org.apache.kafka", name = "kafka_2.11"),
   ExclusionRule(organization = "org.scala-lang.modules", name = "scala-parser-combinators_2.11")
 )
 
+val excludedDeps = excludedDepsNonSpark ++ Seq(
```

Comment:
The following is going to be a bit faster (prepend to a head) and shorter code-wise?

```
val excludedDeps =
    ExclusionRule(organization = "org.apache.spark") +: excludedDepsNonSpark
```

Reply:
Updated in commit https://github.com/splicemachine/splice-machine-spark-connector/pull/32/commits/b79cb27c9e5322b686b9d72373251768e004d420